src/main/java/com/training/tinyurl/dto/UpdateUrlDto.java

```diff
@@ -0,0 +1,20 @@
+package com.training.tinyurl.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UpdateUrlDto {
+
+    @NotBlank(message = "tinyUrl can not be blank")
```

Comment:
What is max length of tiny url that will be generated ?
How much length your client should expect if he is storing your tiny urls in his database if using your API as a library ?